src/attrs.rs

```diff
@@ -64,7 +64,7 @@ fn is_relevant_trait(item: &TraitItem) -> bool {
 }
 
 fn is_relevant_block(block: &Block) -> bool {
-    for stmt in block.stmts.iter() {
+    for stmt in &block.stmts {
```

Comment:
Exactly what I'm trying to do, it's not finding the lib -- I think I need to build it specially